frontend/tutor-milton/src/App.js

```diff
@@ -1,23 +1,11 @@
 import logo from './logo.svg';
 import './App.css';
+import LandingPage from './components/LandingPage';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
```

Comment:
We're going to want to replace the <a> tags with react router tags at some point.